### PR TITLE
Blog post featuring XLAB on provider operations with Ansible

### PR DESCRIPTION
**Pull request by cybette**
@JPrause @Fryguy please take a look

### CODE REVIEW

File: site/_posts/2019-01-09-manageiq-provider-operations-with-ansible-xlab.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## [Dynamic UI for ManageIQ Provider Operations](https://tech.xlab.si/posts/nuage-react-component/)
+
+This second post describes how mind-blowingly powerful it is to implement frontend for provider operations using React Components. These live in a provider codebase and are totally independent of the ManageIQ UI. You'll learn about this new era of ManageIQ frontend where you can define your own UIs without affecting the core.
+
+## [Ansible Playbook as an Event Callback in CloudForms](https://tech.xlab.si/posts/nuage-ansible-in-automate/)
+
+The last post in the series shows how easy it is to run Ansible playbooks triggered by events that are routed through the ManageIQ Automate engine. This post focuses on acquiring the data required by the playbook to actually invoke the operation against the correct inventory item. These data include the event payload and provider's credentials. To show you the complete process, it includes a video with a step-by-step tutorial.
+
+Enjoy the read! You may contact XLAB at sddc&#64;xlab.si if you have any questions. Or as they like to say... **Get IT done.**
```
Comment (JPrause) [inline site/_posts/2019-01-09-manageiq-provider-operations-with-ansible-xlab.md]: +1 LGTM
Hmm,,..I thought the expression was "Get ur done". :smile:

**Comment by cybette** [comment]
Please don't merge yet, waiting for feedback from XLAB

File: site/_posts/2019-01-09-manageiq-provider-operations-with-ansible-xlab.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+author: cybette
+date: 2019-01-09
+comments: true
+published: true
+tags: collaboration tutorials
+---
+
+[Happy 2019](https://www.xlab.si/happy2019/)! We kick off this year with a great feature on collaboration.
+
+In ManageIQ we introduced a model for adding provider features using [Ansible](https://www.ansible.com). Our friends at [XLAB](https://xlab.si) were the first to work with this new technology enablement integrating some cool operations for the [Nuage](http://www.nuagenetworks.net) SDN provider. In this endeavour they used Ansible to implement the provider's CRUD operations through a set of Ansible roles that are available on [Galaxy](https://galaxy.ansible.com/xlab_si). To support these new operations they also created new UI forms using [React.js](https://reactjs.org). These forms are completely tailored to the specific needs of this provider.
```
Comment (gberginc) [inline site/_posts/2019-01-09-manageiq-provider-operations-with-ansible-xlab.md]:
```suggestion
In ManageIQ we introduced a model for adding provider features using [Ansible](https://www.ansible.com). Steampunk team at [XLAB](https://steampunk.si) was the first to work with this new technology enablement integrating some cool operations for the [Nuage](http://www.nuagenetworks.net) SDN provider. In this endeavour they used Ansible to implement the provider's CRUD operations through a set of Ansible roles that are available on [Galaxy](https://galaxy.ansible.com/xlab_si). To support these new operations they also created new UI forms using [React.js](https://reactjs.org). These forms are completely tailored to the specific needs of this provider.
```

File: site/_posts/2019-01-09-manageiq-provider-operations-with-ansible-xlab.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## [Dynamic UI for ManageIQ Provider Operations](https://tech.xlab.si/posts/nuage-react-component/)
+
+This second post describes how mind-blowingly powerful it is to implement frontend for provider operations using React Components. These live in a provider codebase and are totally independent of the ManageIQ UI. You'll learn about this new era of ManageIQ frontend where you can define your own UIs without affecting the core.
+
+## [Ansible Playbook as an Event Callback in CloudForms](https://tech.xlab.si/posts/nuage-ansible-in-automate/)
+
+The last post in the series shows how easy it is to run Ansible playbooks triggered by events that are routed through the ManageIQ Automate engine. This post focuses on acquiring the data required by the playbook to actually invoke the operation against the correct inventory item. These data include the event payload and provider's credentials. To show you the complete process, it includes a video with a step-by-step tutorial.
+
+Enjoy the read! You may contact XLAB at sddc&#64;xlab.si if you have any questions. Or as they like to say... **Get IT done.**
```
Comment (gberginc) [inline site/_posts/2019-01-09-manageiq-provider-operations-with-ansible-xlab.md]:
```suggestion
Enjoy the read! You may contact XLAB Steampunkers at steampunk&#64;xlab.si if you have any questions. Or as they like to say... **Get IT done.**
```

**Comment by Fryguy** [comment]
Looks great! I've been reading the XLAB blog posts as they've been coming out, and they are really awesome, so I'm glad to see us featuring them on the ManageIQ blog. :+1:

**Comment by cybette** [comment]
@Fryguy all credits go to XLAB, even the idea of the post :smile:

@gberginc @miha-plesko please confirm the changes are ok. Thanks for your feedback!

**Pull request merged**